### PR TITLE
Update notifier deps

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

File: notifier/import_map.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  "imports": {
+    "envalid": "https://deno.land/x/envalid@0.1.2/mod.ts",
+    "grammy": "https://deno.land/x/grammy@v1.11.2/mod.ts",
+    "oak": "https://deno.land/x/oak@v11.1.0/mod.ts",
+    "oak/": "https://deno.land/x/oak@v11.1.0/",
+    "octokit/": "https://raw.githubusercontent.com/octokit/webhooks-methods.js/v2.0.0/src/",
+    "postgres": "https://deno.land/x/postgres@v0.17.0/mod.ts",
+    "std/": "https://deno.land/std@0.164.0/"
+  }
+}
```
Comment (KnorpelSenf) [inline notifier/import_map.json:11]: Consider being consistent here by never including the file path to entrypoints in the import map.

**Review by KnorpelSenf — APPROVED** [review]
LGTM, just a suggestion that is fully optional:

File: notifier/import_map.json (hunk 1 above)
Comment (roj1512) [reply notifier/import_map.json:11]: Do you by entrypoint mean "/"?

Comment (roj1512) [reply notifier/import_map.json:11]: If I don't do "std/", neither "testing/", I will waste minutes in thinking of a good name for "testing/asserts.ts".

Comment (KnorpelSenf) [reply notifier/import_map.json:11]: I mean that `/mod.ts` could be moved to the import statements, rather than being hard-coded here.

**Review by KnorpelSenf — APPROVED** [review]
so pretty, such wow

**Pull request merged**